src/thx/Arrays.hx

```diff
@@ -1160,6 +1161,17 @@ Finds the min element of the array given the specified ordering.
 
     return Options.toFailure(Nel.fromArray(collisions), m);
   }
+
+  public static function toStringMap<V>(arr: ReadonlyArray<Tuple<String, V>>): StringMap<V> {
```

Comment:
Yeah, I can't make it work because of lack of covariance: `src/thx/schema/SchemaDynamicExtensions.hx:50: characters 6-10 : thx.Validation<thx.Nel<thx.schema.ParseError>, haxe.ds.StringMap<MapSchema.B>> should be thx.VNel<thx.schema.ParseError, Map<String, MapSchema.B>>`